### PR TITLE
refine docs, code formatting

### PR DESCRIPTION
**Pull request by yungez**
refine docs and code formatting before release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This extension is to help user use [ansible](https://www.ansible.com/) in VSCode
   - Syntax highlighting.
   - Run playbook from Terminal.
     - On Windows, run ansible inside docker.
```
Comment (zikalino) [inline README.md]: If you changed "provide" -> "provides" in the line below, so perhas here "run" -> "runs" as well?
also "ansible" -> Ansible

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # Visual Studio Code extension for Ansible
 
 ## Overview
-This extension is to help user use [ansible](https://www.ansible.com/) in VSCode.
+This extension is to help user use [Ansible](https://www.ansible.com/) in VSCode.
```
Comment (zikalino) [inline README.md]: I think "user use" doesn't sound well. 
Also "This extension is to help" -> "This extension helps ..."

Or maybe it should be sth like: "This extension makes it easy to edit and execute Ansible playbooks from within Visual Studio Code"? just thinking

File: README.md [hunk 3]
```diff
@@ -34,22 +34,22 @@ This extension is to help user use [ansible](https://www.ansible.com/) in VSCode
   
   - Run Ansible Playbook in Terminal
     1. Input playbook file full path, or use default one.
-    1. This step is optinal. If you'll run ansible cloud provider modules, you'll need set cloud provider credentials in credential yaml file, default path is $HOME/.vscode/ansible-credentials.yml, or change credential file path by settings item  `ansible.credentialsFile`. Credential file template is at [here](https://github.com/VSChina/vscode-ansible/blob/readme1/config/credentials.yml).
+    1. This step is optinal. If you'll run ansible cloud provider modules, you'll need set cloud provider credentials in credential yaml file, default path is $HOME/.vscode/ansible-credentials.yml, or change credential file path by settings item  `ansible.credentialsFile`. Credential file template is at [here](https://github.com/VSChina/vscode-ansible/blob/master/config/credentials.yml).
     1. On Non-Windows platform, choose option `docker` or `local`.
```
Comment (zikalino) [inline README.md]: i just see 1. 1. 1. here :-)

File: README.md (hunk 2 above)
Comment (yungez) [reply README.md]: nice, fixed

File: README.md (hunk 1 above)
Comment (yungez) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (yungez) [reply README.md]: it's markdown syntax

File: README.md [hunk 4]
```diff
@@ -4,16 +4,16 @@
 # Visual Studio Code extension for Ansible
 
 ## Overview
-This extension is to help user use [Ansible](https://www.ansible.com/) in VSCode.
+This extension helps use [Ansible](https://www.ansible.com/) efficinently in VSCode.
```
Comment (zikalino) [inline README.md]: "helps to use"
"efficinently" -> "efficiently"

**Pull request merged**